tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "noImplicitAny": false,
     "moduleResolution": "node",
     "allowJs": true,
-    "typeRoots": ["../../node_modules/@types", "./@types"],
+    "typeRoots": ["node_modules/@types", "../../node_modules/@types", "./@types"],
```

Comment:
This prevents issues with typescript having conflicts with `@types` definitions on the root folder and on the test project directories.